assets/app/views/create/fromimage.html

```diff
@@ -1,222 +1,225 @@
-      <div ng-controller="CreateFromImageController" class="create-from-image">
-        <div class="row">
-          <div class="col-md-2 template-name">
-            <span class="fa fa-cubes hidden-xs hidden-sm"></span>
-          </div>
-          <div class="col-md-8">
-            <osc-image-summary resource="image" name="imageName"></osc-image-summary>
-            <form class="" ng-show="imageRepo" novalidate name="form">
-              <div>
-                  <h2>
-                    Name
-                  </h2>
-                <div class="form-group" ng-class="{'has-error': form.appname.$error.oscResourceNameValidator}">
-                 <input class="form-control"
-                        type="text" ng-model="name" name="appname" required="true" osc-resource-name-validator>
-                </div>
-                <p class="help-block">Used to uniquely identify within this project all the resources created to support the application.</p>
-                <div class="has-error" ng-show="form.appname.$error.oscResourceNameValidator">
-                  <span class="help-block"><strong>Please enter a valid name.</strong>
-                  <p>A valid name is applied to all generated resources and is an alphanumeric (a-z, and 0-9) string, with a maximum length of 24 characters, with the '-' character allowed anywhere except the first or last character.</p>
-                  </span>
-                </div>
-                <div class="has-error" ng-show="nameTaken">
-                  <span class="help-block">This name is already in use within the project. Please choose a different name.</span>
-                </div>
-              </div>
-
-              <osc-form-section
-                header="Routing" 
-                about-title="Routing"
-                about="Routing is a way to make your application publicly visible. Otherwise you may only be able to access your application by it's IP address, if allowed by the system administrator."
-                >
-                <div ng-hide="$parent.expand">
-                  <div>
-                    <label>Create a route to the application: </label>
-                    <span>{{routing | yesNo}}</span>
-                  </div>
-                </div>
-                <div ng-show="$parent.expand">
-                  <div class="form-group">
-                    <label class="checkbox-inline">
-                      <input type="checkbox" ng-model="routing.include">
-                        Create a route to the application
-                        <span class="help action-inline">
-                          <i class="pficon pficon-help" data-toggle="tooltip" data-placement="right" data-original-title="Additional routes with alternative configuration options (security, TLS termination, etc) can be added using the osc command."></i>
-                        </span>
-                    </label>
+<div ng-controller="ProjectController">
```

Comment:
Moved this `div` to the top to be consistent other templates and avoid this style, which caused problems once we added the return to overview link.

```
  #content-wrap > .container {
    margin-top: 35px;
    margin-bottom: @grid-gutter-width * 2;
    h1 {
      margin-top: 10px;
    }
  }
```